misc/l10n_testing.md

```diff
@@ -12,46 +12,114 @@ Other docs will cover how the Mozilla approach to the first two stages of the re
 Each Mozilla project has its own process for taking your translations and making them available for you to test. Generally speaking, the process moves like this:
 1. You submit a translation through Pontoon.
 2. The translation goes through a linguistic review and a technical review.
-3. Once approved, the translation is automatically pushed into the project's repository.
-4. Depending on the project, you'll need to wait anywhere from 30 minutes to 24 hours to look for your translation in the Mozilla project.
-5. Once you've found your translation, if you find a bug with it in the project, you return to Pontoon to make any necessary corrections.
+3. Once approved, the translation is automatically pushed into the project’s repository.
+4. Depending on the project, you’ll need to wait anywhere from 30 minutes to 24 hours to look for your translation in the Mozilla project.
+5. Once you’ve found your translation, if you see a problem with it, you return to Pontoon to make any necessary corrections.
 
-Pontoon simplifies these steps for some Web projects by enabling the in-context editor (aka WYSIWYG editor). This allows you to see your translations in-context the moment you create them in Pontoon, cutting out the 30 minutes to 24 hour waiting period to test.
+Pontoon simplifies these steps for some Web projects by enabling the in-context editor (aka WYSIWYG editor). This allows you to see your translations in-context the moment you create them in Pontoon, cutting out the 30 minutes to 24 hours waiting period to test. Most projects do not have this enabled, however, so it is still important to know the typical testing process for your projects.
 
-You might be wondering, "what types of bugs am I supposed to be looking for when I test?" Good question :-) Here's a list of common l10n bugs with screenshots that illustrate the problems in context.
+You might be wondering, “what sort of problems am I supposed to be looking for when I test?” Good question :-) Here’s a list of common l10n bugs with screenshots that illustrate the problems in context.
 
 ## Common l10n bugs
 
 ### Encoding and fonts
 
+Character encoding errors occur when the browser can’t find the correct symbol to display for a character. Normally, you will recognize these errors when you see Unicode replacement characters (�) instead of letters or radicals in your language. These can often be resolved by either changing your browser’s default encoding or by asking developers to use a Unicode-enabled font on the site.
```

Comment:
I think this paragraph is mixing encoding and font support in a strange way.

The first part is about encoding issues, but then you talk about fonts in the end. I would change the last sentence to something like this:

> In some cases the problem can be resolved by changing your browser’s default encoding, in some cases the problem is due to the website not serving content with the right encoding and can be fixed only by the website’s owner.

The problem you're showing in the image is the font not having support for a character, which is a different problem (font support for the language's specific glyphs, not encoding).